src/joint_state_listener.cpp

```diff
@@ -123,7 +127,7 @@ void JointStateListener::callbackJointState(const JointStateConstPtr& state)
 
   // determine least recently published joint
   ros::Time last_published = now;
-  for (unsigned int i=0; i<state->name.size(); i++) {
+  for (unsigned int i = 0; i < state->name.size(); i++) {
```

Comment:
Since we are changing this line, we should probably just make i a `size_t` to be most correct.

Reply:
size_t in d780506c444f1519c78c29717b7f4296a884b016